api/src/test/java/org/openmrs/module/fhir2/api/impl/FhirPersonServiceImplTest.java

```diff
@@ -79,7 +79,7 @@
 	private org.openmrs.Person person;
 	
 	@Mock
-	private FhirPersonDao dao;
+	private FhirPersonDao fhirPersonDao;
```

Comment:
Why did you rename this?

Reply:
This is a vairable except if am to change it where it was used. but by default it was like that

Reply:
I'd just leave this as `dao`.